Library/Formula/sqlite-analyzer.rb

```diff
@@ -0,0 +1,19 @@
+class SqliteAnalyzer < Formula
+  homepage "https://www.sqlite.org/"
+  url "https://www.sqlite.org/2015/sqlite-src-3080803.zip"
+  version "3.8.8.3"
+  sha256 "790ff6be164488d176b3bed7e0e0850bac1567a4011381307685d48eb69fab48"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--prefix=#{prefix}"
```

Comment:
Since there are only two options, you might be able to get away with squeezing them both onto the same line rather than having to split across multiple lines.